data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -334,6 +334,33 @@ sed -i 's/CR/\n/g' /root/.ssh/id_rsa
 chmod 600 /root/.ssh/id_rsa
 echo <%= $get_var->('_SECRET_RSA_PUB_KEY') %> > /root/.ssh/id_rsa.pub
 echo <%= $get_var->('_SECRET_RSA_PUB_KEY') %> >> /root/.ssh/authorized_keys
+]]>
+        </source>
+      </script>
+      <script>
+        <!-- enable persistent kernel log as per ENABLE_PERSISTENT_KERNEL_LOG -->
+        <filename>enable_persistent_kernel_log.sh</filename>
+        <source><![CDATA[
+% if ($get_var->('ENABLE_PERSISTENT_KERNEL_LOG')) {
+    config_file="/etc/rsyslog.conf"
+    if [ -f $ssh_config_file ];then
```

Comment:
Is it `$ssh_config_file` or typo of `$config_file`?

Reply:
Done